### DISCUSSION TITLE
CANARY BROWSER AND DRIVER

### DISCUSSION THREAD

**Discussion by Mayurlambdatest** [SpyNetGirl/MSEdgeFeatures#5] [General]
My query is regarding the canary browser implementation

I have a small doubt related to the browser version and driver version of canary where the new launches of canary browser is dependent on the new drivers of the same tag ?

Please do let me know whether is a superficial need or x-1 driver version may work for the newly launched browser version

**Comment by HotCakeX** [comment]
Hi @Mayurlambdatest 

I'm sorry but could you clarify what you mean by driver?
